tests/UlidTest.php

```diff
@@ -116,7 +153,7 @@ public function testCreatesFromStringWithInvalidAlphabet($ulid): void
 
     public function testConvertsToTimestamp(): void
     {
-        $this->assertEquals(1561622862, Ulid::fromString('0001EH8YAEP8CXP4AMWCHHDBHJ')->toTimestamp());
```

Comment:
In my view these timestamps are perfectly valid and try to understand why you and issue #23 are saying that they are not. Could you elaborate on that?